fenix/app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -327,6 +329,10 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
                     Events.appOpened.record(Events.AppOpenedExtra(it))
                     // This will record an event in Nimbus' internal event store. Used for behavioral targeting
                     components.analytics.experiments.recordEvent("app_opened")
+
+                    if (safeIntent.action.equals(ACTION_OPEN_PRIVATE_TAB) && it == "APP_ICON") {
```

Comment:
🔍 nit: Should we refactor "APP_ICON" and "LINK" into constants since we're not using it in more than one place?

Reply:
refactored! good call